auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -310,12 +311,9 @@ import { KCUserProfile } from 'sbc-common-components/src/models/KCUserProfile'
 import LinkedBCOLBanner from '@/components/auth/common/LinkedBCOLBanner.vue'
 import ModalDialog from '../../common/ModalDialog.vue'
 import OrgAdminContact from '@/components/auth/account-settings/account-info/OrgAdminContact.vue'
-
-import { namespace } from 'vuex-class'
-
-const CodesModule = namespace('codes')
-const OrgModule = namespace('org')
-const userModule = namespace('user')
+import { useCodesStore } from '@/stores/codes'
+import { useOrgStore } from '@/stores/org'
+import { useUserStore } from '@/stores/user'
```

Comment:
Travis, one nice change you can also make is export the stores in the `index.ts` file. In doing that, you can import all the stores easily. For example, here it would be:

`import { useCodesStore, useOrgStore, useUserStore } from '@/stores'.` 

This'll also make life easier in case an additional store is to be added in the future. 

However, this'll take time to do and I think you just want this PR through at this point. So, just a suggestion for later work maybe.

Reply:
Yup, was done after the fact - there was issues getting Vuex to work

Reply:
Alright, sounds good Travis!